Consul.Test/OperatorTest.cs

```diff
@@ -89,5 +89,12 @@ public async Task Operator_GetLicense()
             var queryResult = await _client.Operator.GetConsulLicense();
             Assert.NotNull(queryResult.Response);
         }
+
+        [EnterpriseOnlyFact]
+        public async Task Segment_List()
+        {
+            var segments = await _client.Operator.SegmentList();
+            Assert.IsType<string[]>(segments.Response);
```

Comment:
Maybe we could check that the response is not empty (Check if the length of the array is non-zero?)

Reply:
Thanks @marcin-krystianc I have made the update